README.md

```diff
@@ -56,12 +56,38 @@ If you would like to use DragsterJS as ES6 module in the browser:
 ```html
 <script type="module">
     import Dragster from './dragster.es6.js';
-
     const dragster = Dragster();
 </script>
 ```
 
-And start having fun with dragging elements on the website.
+These few lines of CSS are required to prevent issues while dropping the elements.
+
+``` css
+/*
+ * Prevents user from selecting text
+ * in the draggable elements
+ */
+[draggable] {
+  -moz-user-select: none;
+  -khtml-user-select: none;
```

Comment:
Do you think, we can strip some of the prefixed properties here and leave just `user-select: none`?

Reply:
Unfortunately this property is not widely supported un-prefixed by browsers according to MDN and CanIUse:
https://developer.mozilla.org/en-US/docs/Web/CSS/user-select
https://caniuse.com/#search=user-select

I left it prefixed for safety.

Reply:
Ok, so let's keep only these prefixes: `-ms-`, `-moz-` and `-webkit-` with `none` value. Other ones could be stripped out of your PR's code.

Reply:
Opera uses the same rendering engine as Chrome, so `-webkit-` would be enough.